core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2729,7 +2920,381 @@ components:
+              format: int64
+              minimum: 0
+              maximum: 10000000000
+              description: |
+                An integer between `0` and `10^10`, marking the epoch from which the transaction will no longer be valid, and be permanently rejected.
+                Only present if the rejection isn't permanent.
+##############################################################
+# REQUEST: /lts/state/account-all-fungible-resource-balances #
+##############################################################
+    LtsStateAccountAllFungibleResourceBalancesRequest:
+      type: object
+      required:
+        - network
+        - account_address
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        account_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the account's address
+    LtsStateAccountAllFungibleResourceBalancesResponse:
+      type: object
+      required:
+        - state_version
+        - account_address
+        - fungible_resource_balances
+      properties:
+        state_version:
+          type: integer
+          format: int64
+          minimum: 1
+          maximum: 100000000000000
+          description: |
+            An integer between `1` and `10^13`, giving the state version at which the account was read.
+        account_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the account's address
+        fungible_resource_balances:
+          description: A list containing all resource balances for the requested account.
+          type: array
+          items:
+            $ref: "#/components/schemas/LtsFungibleResourceBalance"
+    LtsFungibleResourceBalance:
+      type: object
+      required:
+        - fungible_resource_address
+        - amount
+      properties:
+        fungible_resource_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the fungible resource's global address
+        amount:
+          type: string
+          description: |
+            The string-encoded decimal representing the amount of the fungible resource.
+            A decimal is formed of some signed integer `m` of attos (`10^(-18)`) units, where `-2^(256 - 1) <= m < 2^(256 - 1)`.
+#########################################################
+# REQUEST: /lts/state/account-fungible-resource-balance #
+#########################################################
+    LtsStateAccountFungibleResourceBalanceRequest:
+      type: object
+      required:
+        - network
+        - account_address
+        - fungible_resource_address
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        account_address:
+          type: string
+          description: The Bech32m-encoded human readable version of the account's address
+        fungible_resource_address:
```

Comment:
I think it's fine / better to just call this `resource_address`, as long as we give a decent error message if they provide a nf resource address.

I care about including accurate naming on our responses, so people don't get confused. I mind less on the inputs.